cmake/Dependencies.cmake

```diff
@@ -244,7 +244,7 @@ endif()
 # ---[ Caffe2 Int8 operators (enabled by USE_QNNPACK) depend on gemmlowp and neon2sse headers
 if(USE_QNNPACK)
   set(CAFFE2_THIRD_PARTY_ROOT "${PROJECT_SOURCE_DIR}/third_party")
-  include_directories(SYSTEM "${CAFFE2_THIRD_PARTY_ROOT}/gemmlowp")
+  include_directories(SYSTEM "${CAFFE2_THIRD_PARTY_ROOT}")
```

Comment:
I think the reason they are done as double-directories is so that we can set an `include_directories` without pulling in *everything* from third-party. So unless this is fixing a real problem, I am not sure we can take this patch.